stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -1260,30 +1238,20 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     }
     
     // slightly faster paths for common sequences
-    
+    @inlinable
```

Comment:
Are you sure this `@inlinable` doesn't expose the `_DataStorage` type?

Reply:
@airspeedswift was telling me this is ok (similar to how Array does it)